var/spack/repos/builtin/packages/libtree/package.py

```diff
@@ -15,9 +15,9 @@ class Libtree(CMakePackage):
 
     maintainers = ['haampie']
 
-    version('1.2.0', sha256='4316a52aed7c8d2f7d2736c935bbda952204be92e56948110a143283764c427c')
-    version('1.1.4', sha256='cfafb24c8f5e0d356c82777c338d58730ca6f3cb76dfe8a6857ee3ad65bf8be7')
-    version('1.1.3', sha256='7baf5aaecd3a076bf1e7a1aa86979e7b841ab3f678ca8ac0e2a22bbbccf0dd06')
+    version('1.2.0', sha256='3e74655f22b1dcc19e8a1b9e7796b8ad44bc37f29e9a99134119e8521e28be97')
+    version('1.1.4', sha256='38648f67c8fa72c3a4a3af2bb254b5fd6989c0f1362387ab298176db5cbbcc4e')
+    version('1.1.3', sha256='4c681d7b67ef3d62f95450fb7eb84e33ff10a3b9db1f7e195b965b2c3c58226b')
```

Comment:
Can you give more insight on why the checksum changed for v1.1.3-v1.2.0 and not for earlier versions?

Reply:
Those versions were submitted at the same time #17035, but the hash was wrongly computed for the sources.tar.gz that github provides, not the custom tarball which includes submodules as well.